platforms/Windows/bundle/installer.wixproj

```diff
@@ -8,7 +8,7 @@
       $(DefineConstants);
       IncludeAndroid=$(Platforms.Contains("android"));
       IncludeWindows=$(Platforms.Contains("windows"));
-      INCLUDE_NOASSERTS=$(INCLUDE_NOASSERTS);
+      Include_NoAsserts=$(ToolchainVariants.Contains("noasserts"));
```

Comment:
We slowly migrated from the `CAPS_UNDERSCORE` to the `TitleCase` variants, where don't have `_`.

Reply:
fixed.